### ISSUE TITLE
[Feature] Ability to show (or not) the BFG Edition's WADs

### ISSUE THREAD

**Issue by silverlays** [ZDoom/gzdoom#2264]
### GZDoom version

GZDoom 4.11.3

### Which game are you running with GZDoom?

None

### What Operating System are you using?

Windows 11

### Is your feature request related to a problem? Please describe.

No

### Describe the solution you'd like

If possible, a simple boolean switch in the configuration file maybe ?

### Add any other context or screenshots about the feature request here.

Since I've installed the BFG Edition of Doom 3 with GoG, my GZDoom list is less intuitive than before. I REALLY don't want to launch my BFG GoG Wad with GZDoom unless i decided it and copied the wads to the gzdoom folder

Thanks in advance.

**Comment by mjr4077au** [comment]
Remove $STEAM from your INI file.

**Comment by silverlays** [comment]
> Remove $STEAM from your INI file.

I don't have it in it :(

**Comment by coelckers** [comment]
GZDoom doesn't have that, it only exists in Raze.

**Comment by mjr4077au** [comment]
> GZDoom doesn't have that, it only exists in Raze.

Ah my bad...

**Comment by JacksonRoberts-Git** [comment]
I might have a solution. I need a project to work on for class and someone on a different issue request said this might be a relatively easy fix. Would you mind assigning me to this?

**Comment by madame-rachelle** [comment]
This seems like a done deal but I'm willing to look at whatever proposal you have in mind.

**Comment by JacksonRoberts-Git** [comment]
I'm new to C++ but the guy in the email said to add a cvar and remove the BFG IWADs from the path if it's set (whatever the hell those mean)

**Comment by JacksonRoberts-Git** [comment]
silverlays, would you be able to show me what your WAD list looks like? I can't make sense of ZDoom's spaghetti code so I was hoping wonderind what it looked like. here's mine.
![Screenshot (42)](https://github.com/user-attachments/assets/2dc88ca3-a640-4eb1-ba32-ddcc688719fd)

**Comment by silverlays** [comment]
Mine looks like this. But unfortunatly, I've uninstalled Doom 3 because of this issue. Just imagine 2 mores lines with « The Ultimate Doom BFG Edition » and « Doom II BFG Edition »
![image](https://github.com/user-attachments/assets/ddc79cfa-709e-482e-b4ed-3e184ee399cb)

**Comment by JacksonRoberts-Git** [comment]
I'm actually not 100% sure what the issue is anymore. I know you want to show the BFG edition's IWAD files but wouldn't they be there in the menu?

**Comment by silverlays** [comment]
Read again, I've edited my previous post 😊 (sorry for that)

**Comment by silverlays** [comment]
Not exactly. My demands is to be able to have a variable in the .ini file to disable/enable the search for IWAD in the Windows registry, like the (Steam/GoG) BFG Edition of Doom 3

**Comment by silverlays** [comment]
The idea is to have a condition for each (or all) of that linked to a variable in the .ini file:

Source: https://github.com/ZDoom/gzdoom/blob/master/src/d_iwad.cpp#L471C1-L473C43
```
mSearchPaths.Append(I_GetGogPaths());
mSearchPaths.Append(I_GetSteamPath());
mSearchPaths.Append(I_GetBethesdaPath());
```

**Comment by JacksonRoberts-Git** [comment]
I have a really dumb idea. Could I just assign searches for that by pressing G, S, and B?

**Comment by silverlays** [comment]
It's a idea indeed, but mine having a boolean in the config file would be worth, don't you think ?

**Comment by JacksonRoberts-Git** [comment]
so it only searches if you activate the boolean? I'm not sure how to set that up, but I'll do everything I can

**Comment by JacksonRoberts-Git** [comment]
I'll probably have to fork the code you sent me instead of the config (as there are multiple config files) but this looks totally doable.

**Comment by Sol1vaN** [comment]
hi,
this still persist
4.12.2 this not happen
4.13.0 begins to show all steam/gog library

i don't want it, i want gzdoom detect only locally my wads an not from libraries.
please, don't forget it.
thnks.
